### PR TITLE
Tom bug clear input field

### PR DESCRIPTION
**Pull request by Tom-Sarfo**
I implemented a way to clear the input field after submitting the form. It is kind of working but I need your review on the solution.

### CODE REVIEW

File: src/CreateTaskForm.js [hunk 1]
```diff
@@ -1,21 +1,12 @@
...
-            setTaskInupt('');//finding it difficult to clear textfield
-            setAsignee('');
-            e.preventDefault();
-             onHandleSubmit(taskInput, asignee);
-            }  
+            onSubmit={e => {onHandleSubmit(e)}  
             }>
-                <input type="text" placeHolder="Enter Task" onChange={e => setTaskInupt(e.target.value)} />
-                <input type="text" placeHolder="Asign Task to" onChange={e => setAsignee(e.target.value)}/>
+                <input type="text" placeHolder="Enter Task" onChange={e => onSetTaskInput(e.target.value)} />
```
Comment (noramens) [inline src/CreateTaskForm.js]: @Tom-Sarfo the attribute here should be`placeholder` not `placeHolder`

File: src/TaskBoard.js [hunk 2]
```diff
@@ -5,10 +5,13 @@ import { useState } from "react"
 
 export default function TaskBoard(){
   const [task, setTask] = useState([]);
+  const [ taskInput, setTaskInupt] = useState('');
+  const [ asignee, setAsignee] = useState('')
   
   
   var taskID = task.length;
```
Comment (noramens) [inline src/TaskBoard.js]: @Tom-Sarfo we had a conversation about declaring variables with `var`

File: src/TaskBoard.js [hunk 3]
```diff
@@ -18,6 +21,10 @@ export default function TaskBoard(){
        done: false
      }
     ]);
+    e.target.reset(); //This works
+    //this is still not working after lifting state
+    // setTaskInupt("");
+    // setAsignee("");
```
Comment (noramens) [inline src/TaskBoard.js]: @Tom-Sarfo always remove commented code before pushing

File: src/TaskBoard.js [hunk 4]
```diff
@@ -45,10 +52,18 @@ export default function TaskBoard(){
 
     return (
         <div className="TaskBoard">
-          <CreateTaskForm onHandleSubmit = { handleFormSubmit } />
+          <CreateTaskForm 
+            onHandleSubmit = { handleFormSubmit }
+            onSetTaskInput = {setTaskInupt}
+            onSetAsignee = {setAsignee}
```
Comment (noramens) [inline src/TaskBoard.js]: @Tom-Sarfo you don't have to add the prefix `on` to every prop you pass when its a function. Usually the `on` prefix is left for handlers

File: src/CreateTaskForm.js (hunk 1 above)
Comment (Tom-Sarfo) [reply src/CreateTaskForm.js]: Okay

File: src/TaskBoard.js (hunk 2 above)
Comment (Tom-Sarfo) [reply src/TaskBoard.js]: Yeah I remember

File: src/TaskBoard.js (hunk 3 above)
Comment (Tom-Sarfo) [reply src/TaskBoard.js]: Okay thank you

File: src/TaskBoard.js (hunk 4 above)
Comment (Tom-Sarfo) [reply src/TaskBoard.js]: Thanks

**Pull request merged**